sequra/database/models.py

```diff
@@ -0,0 +1,118 @@
+import decimal
+
+from sqlalchemy import extract
+from sqlalchemy.orm import validates
+
+from sequra.database import db
+
+
+@validates('email')
+def validate_email(key, address):
+    """ Further work, integration with online email verification service"""
+    assert '@' in address
+    return address
+
+
+class Disbursement(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    _amount = db.Column("amount", db.String(50))
```

Comment:
`amount` is ok for this limited scope, however it may be too ambiguous for future evolution. What other names could we use here? What information may be also useful (related to this calculation)?